packages/nextjs/next.config.mjs

```diff
@@ -11,6 +11,11 @@ const nextConfig = {
         hostname: "identicon.starknet.id",
         pathname: "/**", // Allows all paths under this domain
       },
+      {
+        protocol: "https",
+        hostname: "img.starkurabu.com",
+        pathname: "/**",
+      },
```

Comment:
Another one, might want to either write in the `README.md` of this config, since Next.JS can be pretty fussy. Just a small foot note will do.

Reply:
Done.